ext/exiv2/extconf.rb

```diff
@@ -20,10 +21,10 @@
 end
 
 if have_macro("EXV_USE_CURL", "exiv2/exv_conf.h")
-  if dir_config("libcurl") == [nil, nil]
+  if dir_config("curl") == [nil, nil]
```

Comment:
This only renames the flag `--with-libcurl-dir` to `--with-curl-dir`, but shouldn't affect compilation. Unless we have some really weird side effect here?

Reply:
Given I've resolved my issues and all fingers are pointing to ruby-build, I'm happy to clsoe this PR

Reply:
How about https://github.com/envato/exiv2/pull/34 instead?